### ISSUE TITLE
latin multiple choice tests

### ISSUE THREAD

**Issue by alysawyer** [mikeizbicki/modulus-magnus-linguae#58]
Do you know where I can access the multiple choice parts of past exams? I was only able to find the free response online.

**Comment by alysawyer** [comment]
there seems to be a sizable amount of online practice questions but I assume we want to evaluate it on the actual tests

**Comment by mikeizbicki** [comment]
Yes, real tests would be best... but upon further reflection I think this is going to be a rather hard path for us to take.  I've just taken a look through an AP practice book, and the exams require a lot of "English knowledge about Latin" that at least our current textbook doesn't have inside of it.  So there's no way I would expect any model/person who's only used this book to do well.  It would be an interesting question to also use an English-Latin grammar to give the model this knowledge (and that would be something useful for many reasons), but that's not obvious to me how to do.

**Comment by alysawyer** [comment]
Yeah that makes sense to me -- I was thinking of training it on the texts that are in the syllabus because I think that some of the contextual knowledge comes from those required readings but I don't think that would be entirely sufficient to give it a reasonable shot at the English knowledge portion
